docs/README.md

```diff
@@ -1 +1,5 @@
-This plugin allows channel export into a human readable format.
+This plugin allows channel export into a human readable format. The Mattermost Incident Response plugin supports channel export. The following artifacts are included in the `.CSV` output file:
+
+- Messages
```

Comment:
The export actually only contains messages, and nothing that's incident response specific (i.e no stages or tasks at this time).

Reply:
Perfect, thank you!